react-native/package.json

```diff
@@ -11,9 +11,13 @@
     "start": "cd demo && yarn && cd .. && yarn link:themes && cd demo && yarn start"
   },
   "dependencies": {
-    "@pxblue/colors": "^1.0.1"
+    "@pxblue/colors": "^2.0.0"
   },
   "devDependencies": {
-    "typescript": "^3.7.2"
+    "typescript": "^3.7.2",
+    "@pxblue/react-native-components": "^2.0.2"
+  },
+  "peerDependencies": {
+    "@pxblue/react-native-components": "^2.0.2"
```

Comment:
This peer dependency would actually need to go in the root package.json file because we publish all of the themes as a single package. These subdirectory package.json files don't actually do anything with respect to the package getting published.